source/fundamentals/serialization/poco.txt

```diff
@@ -610,6 +608,58 @@ registering the class map:
    The ``DateTimeKind`` enum is part of the {+framework+}. For more information on
    its members, see the `Microsoft documentation for the DateTimeKind enum. <https://learn.microsoft.com/en-us/dotnet/api/system.datetimekind?view=net-8.0>`__
 
+.. _csharp-poco-dateonly-attr:
+
+Custom DateOnly Serialization
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To customize how the {+driver-short+} serializes `DateOnly
+<https://learn.microsoft.com/en-us/dotnet/api/system.dateonly?view=net-8.0>`__
+properties, use the ``[BsonDateOnlyOptions()]`` attribute and specify
+the following settings:
+
+- ``Representation``: Set to a ``BsonType`` instance that specifies how
+  the ``DateOnly`` value is stored in MongoDB.
+
+- ``DocumentFormat``: Set only if you set the ``Representation``
```

Comment:
Maybe we can say something like "Sets the format to use for the document if `Representation` is `BsonType.Document`, ignored otherwise.